spec/acceptance/1_class_spec.rb

```diff
@@ -93,6 +93,39 @@ class { 'keycloak':
     end
   end
 
+  context 'default with JDBC_PING, clustered mode and postgresql datasource' do
+    it 'runs successfully' do
+      pp = <<-EOS
+      include postgresql::server
+      class { 'keycloak':
+        datasource_driver => 'postgresql',
+        enable_jdbc_ping  => true,
+        operating_mode    => 'clustered',
+      }
+      EOS
+
+      apply_manifest(pp, catch_failures: true)
+      apply_manifest(pp, catch_changes: true)
+    end
+
+    describe service('keycloak') do
+      it { is_expected.to be_enabled }
+      it { is_expected.to be_running }
+    end
+
+    describe port(8080) do
+      it { is_expected.to be_listening.on('0.0.0.0').with('tcp') }
+    end
+
+    describe port(9990) do
+      it { is_expected.to be_listening.on('127.0.0.1').with('tcp') }
+    end
+
+    describe port(7600) do
+      it { is_expected.to be_listening.on('0.0.0.0').with('tcp') }
```

Comment:
This is failing. I am guessing because in the config template you set to `::ipaddress` and here you are looking for `0.0.0.0` so the values don't match.